CHANGELOG.v2.alpha.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.96.1-alpha.0](https://github.com/aws/aws-cdk/compare/v2.96.0-alpha.0...v2.96.1-alpha.0) (2023-09-14)
```

Comment:
Should this be `##`?

Reply:
🤦‍♂️ fixed this but forgot to push the commit - thanks for the catch @colifran !!